src/doc/src/reference/manifest.md

```diff
@@ -244,9 +244,13 @@ package.
 ```toml
 [package]
 # ...
-homepage = "https://serde.rs/"
+homepage = "https://serde.rs"
```

Comment:
Deleted trailing slash to match the real serde. https://github.com/serde-rs/serde/blob/v1.0.195/serde/Cargo.toml#L10